src/components/pagination/demo/mock-pagination-data.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * These functions mimic the structure of WordPress's pagination objects.
+ *
+ * @see https://developer.wordpress.org/reference/functions/paginate_links/
+ * @see https://timber.github.io/docs/guides/pagination/
+ */
+
+function mockPageData(page = 1, current = 1) {
+  return {
+    title: page,
+    text: page,
+    name: page,
+    current: page === current,
+    link: page !== current && `#page-${page}`,
+  };
+}
+
+export default function mockPaginationData({
```

Comment:
Could you add JSDoc-style comment blocks here? It makes reading JS files a lot easier when functions have a clear name + description comment block.

Also explaining the options would be helpful. I've read the function and I still have no idea what `midSize` is.

Reply:
I've tended toward being pretty lean with demonstrative or prototype JavaScript since it's not intended for production.

I had hoped that linking to two pages that describe what `mid_size` is at the top of the file would suffice.

Reply:
Ah, I didn't understand that this isn't production JS. That makes more sense, then.

Reply:
- [x] Added JSDoc-style comments for this file